conda_build/metadata.py

```diff
@@ -105,7 +105,11 @@ def ns_cfg(config):
                     py33=bool(py == 33),
                     py34=bool(py == 34),
                     py35=bool(py == 35),
-                    py36=bool(py == 36),))
+                    py36=bool(py == 36),
```

Comment:
We have made an explicit decision to not extend this going forward.  This PR will not be accepted with this change.

Reply:
Removed.